mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml

```diff
@@ -0,0 +1,396 @@
+                            name="actions"
+                        >
+                        <separator string="Action Plan" />
+                        <field
+                                name="action_ids"
+                                attrs="{'readonly':[('state','not in',['pending'])]}"
+                            />
+
+                        <group string="Plan Review">
+                            <group>
+                                <field
+                                        name="action_comments"
+                                        attrs="{'readonly':[('state','not in',['pending'])]}"
+                                        nolabel="1"
+                                        placeholder="Action Plan Comments"
+                                    />
+                            </group>
+                        </group>
+
+                        <group
+                                attrs="{'invisible':[('state','not in',['open','done'])]}"
+                                string="Effectiveness Review"
+                            >
+                            <group>
+                                <field
+                                        name="evaluation_comments"
+                                        attrs="{'readonly':[('state','not in',['open'])]}"
+                                        nolabel="1"
+                                        placeholder="Evaluation Comments"
+                                    />
+                            </group>
+                        </group>
+                    </page>
+
+                </notebook>
+            </sheet>
+
+            <div class="oe_chatter">
+                <field
+                        name="message_follower_ids"
+                        widget="mail_followers"
+                        groups="base.group_user"
+                    />
+                <field name="activity_ids" widget="mail_activity" />
+                <field name="message_ids" widget="mail_thread" />
+            </div>
+
+        </form>
+    </field>
+</record>
+
+    <record model="ir.actions.act_window" id="open_mgmtsystem_nonconformity_list">
+        <field name="name">Nonconformities</field>
+        <field name="res_model">mgmtsystem.nonconformity</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="view_id" ref="view_mgmtsystem_nonconformity_kanban" />
+        <field name="search_view_id" ref="view_mgmtsystem_nonconformity_filter" />
+        <field name="context">{"search_default_user_id":uid}</field>
+    </record>
+
+    <!-- <menuitem -->
+    <!--     id="menu_open_nonconformity" -->
+    <!--     string="Nonconformity" -->
+    <!--     action="open_mgmtsystem_nonconformity_list" -->
+    <!--     parent="mgmtsystem.menu_mgmtsystem_main" -->
+    <!--     sequence="30" -->
+    <!--     groups="mgmtsystem.group_mgmtsystem_manager,mgmtsystem.group_mgmtsystem_auditor,base.group_user"/> -->
```

Comment:
Unless I am missing something, without this menuitem, there is no way to access the nonconformity view from the menu. I think the 'string' attribute is causing an error when menuitem is uncommented.
```suggestion
    <menuitem
         id="menu_open_nonconformity"
         name="Nonconformities"
         action="open_mgmtsystem_nonconformity_list"
         parent="mgmtsystem.menu_mgmtsystem_main"
         sequence="30"
         groups="mgmtsystem.group_mgmtsystem_manager,mgmtsystem.group_mgmtsystem_auditor,base.group_user"/>
```